lib/assert.js

```diff
@@ -314,21 +320,29 @@ function setEquiv(a, b, strict, memo) {
   if (a.size !== b.size)
     return false;
 
+  // This is a set of the entries in b which have been consumed in our pairwise
+  // comparison. Initialized lazily so sets which only have value types can
+  // skip an extra allocation.
+  let bEntriesUsed = null;
```

Comment:
Sure - I don't have a strong opinion so I've renamed it.

My concern with `usedEntries` is that it doesn't make it clear that we're marking entries in set `b`.